client/lib/abtest/active-tests.js

```diff
@@ -123,4 +123,13 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: true,
 	},
+	redirectToCustomerHome: {
+		datestamp: '20191220',
+		variations: {
+			variant: 10,
+			control: 90,
+		},
+		defaultVariation: 'control',
+		allowExistingUsers: true,
```

Comment:
Should we limit to new users so existing users aren't confuzzled?

Reply:
Dang, thanks for picking that up @jblz 

I flew over the description in p58i-89H-p2, not registering that it would be a follow up test.

> Test 10% of all users instead of 10% of new users

Will fix.